src/components/App/App.tsx

```diff
@@ -34,6 +34,9 @@ const ErrorPage = lazy(() => import('../404Page'));
 const VoucherRedemptionPage = lazy(() =>
   import('../../pages/VoucherRedemption')
 );
+const MerchantGiftCardsPage = lazy(() =>
```

Comment:
I'd rather we name this MerchantVouchersPage to keep it consistent with VoucherRedemptionPage, but up to you!

Reply:
Makes sense to me! I started with Gift Cards as that's what the endpoint is called, but then I realized we're showing 'Voucher' everywhere. I might change it to MerchantVoucherDashboard actually